randovania/games/prime2/json_data/Agon Wastes.txt

```diff
@@ -1492,9 +1498,13 @@ Extra - in_dark_aether: False
           All of the following:
               Space Jump Boots
               Any of the following:
-                  After Central Mining Station Pirates
-                  Scan Visor and Combat/Scan Dash (Advanced)
+                  After Central Mining Station Pirates or Use Screw Attack (Space Jump)
                   Morph Ball and Roll Jump (Advanced)
+                  All of the following:
+                      Scan Visor
+                      All of the following:
+                          # https://www.youtube.com/watch?v=bHhacf7YEQY
+                          Extended Dash (Expert)
```

Comment:
Since the only thing nested under the Scan Visor path is the Extended Dash, just put the trick with the item and list the video together. Regardless, this needs to be moved out of this section since this is all put under the SJ group.

Reply:
Done.